backend/uclapi/common/cachet.py

```diff
@@ -55,97 +57,120 @@ def get_incident_name(base: str) -> str:
     return incident_name
 
 
-def create_incident(error_message: str, component_name: str):
+def create_incident(error_message: str, component_name: str,
+                    incident_status: int, comp_status: int):
     """
-    Create an incident with the error message specified for the component
-    name specified. This marks the status as 4 - major outage - and the
-    incident status as 1 - investigating.
+    Create an incident with the error message and status specified for the
+    component with its name and new status specified. The status is specified
+    by the
+    cachet.enums enum. Only creates an incident if the component isn't
```

Comment:
No it doesn't, its custom logic since in a normal prod environment you might want to create multiple instances for say, database connection failure and just a general database error, whereas we only care about did it work or did it not.